kitsune/products/views.py

```diff
@@ -45,14 +57,33 @@ def product_landing(request, slug):
             latest_version = versions[0].min_version
         else:
             latest_version = 0
+    topics = topics_for(request.user, product=product, parent=None)
+    # Create a dictionary of topics and three documents
+    # TODO: get popular documents instead of just the first three
```

Comment:
Can't we use the `get_featured_articles` from wiki/utils (with small changes) to get the most popular docs for a specific topic and product combo?

Reply:
Being super atomic, this was the look/feel until we had the details for exactly what to show from content. I don't mind adding the top docs as the default though.

Reply:
Done. I modified `get_featured_articles` to accept a topic.